scripts/parser-tests/test262/allowlist.txt

```diff
@@ -1 +1,16 @@
-
+language/module-code/import-assertion-key-identifiername.js(default)
```

Comment:
There tests fail because we currently throw `The only accepted module attribute is `type`. We might want to relax that restriction.